### ISSUE TITLE
Make clear how this repo relates to govuk-country-and-territory-autocomplete

### ISSUE THREAD

**Issue by edwardhorsford** [alphagov/accessible-autocomplete#224]
I don't think it's clear how this repo relates to @alphagov/govuk-country-and-territory-autocomplete . Both should be clear.

I've come across two cases of GDS teams using the accessible autocomplete when they probably wanted to be use the country and territory autocomplete. In both cases it seems the teams were not aware of the country and territory autocomplete. This is a massive shame as much of our work / user research / ui tweaks are in the other repo, not this one.

Can we make the differences and the link between them clearer?

**Comment by vanitabarrett** [comment]
There is a similar card in the govuk-country-and-territory-picker repo to make the link with the accessible autocomplete clear. It would make sense to look at both of these together. https://github.com/alphagov/govuk-country-and-territory-autocomplete/issues/23

**Comment by EoinShaughnessy** [comment]
Just wondering about the difference between the autocompletes - is the accessible autocomplete for everything else not covered by the country-and-territory autocomplete?

Re fixing this issue and [#23](https://github.com/alphagov/govuk-country-and-territory-autocomplete/issues/23) - maybe we could:

- be explicit in each README about what the repo at hand is for
- add links from each README to the README for the other autocomplete

**Comment by edwardhorsford** [comment]
@EoinShaughnessy 

There might be an argument for a longterm plan to merge them.

The history of the difference is that we didn't start building a generic component, we started with the country picker - as that was by far the common need for an autocomplete. So by doing a country picker we would get the autocomplete for free.

We did all of our development, design, and user research using the country picker. Most of the deisgn optimisations happened in it. We also made several optimisations specific for picking countries - more complex synonym support and way that hints work.

A subset of the codebase was then forked out to make the accessible autocomplete. We intended that some of the better features from the country picker would be reintroduced, but this sadly never happened. In particular the autocomplete ships with a really naive matching algorithm and no sorting algorithm.
